pkg/arvo/app/ping.hoon

```diff
@@ -406,62 +154,33 @@
 ++  on-agent
   |=  [=wire =sign:agent:gall]
   ^-  [(list card) _this]
-  =^  cards  state
-    ?+    wire  `state
-        [%nat *]
-      ?.  ?=(%nat -.plan.state)  `state
-      ?.  ?=(%poke-ack -.sign)   `state
-      (take-ping:nat now.bowl wire p.sign)
-    ::
-        [%pub *]
-      ?.  ?=(%pub -.plan.state)  `state
-      ?.  ?=(%poke-ack -.sign)   `state
-      (take-pings:pub wire p.sign)
-    ::
-        [%one *]
-      ?.  ?=(%one -.plan.state)  `state
-      ?:  ?=(%poke-ack -.sign)   `state
-      :: XX handle error?
-      `state
-    ==
-  [cards this]
+  ?.  ?=([%ping s=@ *] wire)
+    `this
+  ?.  =(galaxy.state (slav %p i.t.wire))
+    `this
```

Comment:
This should be fine, so long as we only ever set the galaxy at the same time was we emit the ping (outside of `+on-init`).